articles/best-practices/testing.md

```diff
@@ -0,0 +1,84 @@
+---
+description: Best practices for testing.
+topics:
+  - best-practices
+  - custom-database
+  - extensibility
+  - database-action-scripts
+  - custom-database-connections
+  - scripts
+  - rules
+contentType: reference
+useCase:
+  - best-practices
+  - custom-database
+  - database-action-scripts
+  - rules
+---
+
+# Testing
```

Comment:
Hmmm. There are numerous references to Rules within the body of this section. In this context, we're talking about Action Scripts not Rules. I had rather envisaged that we might link Rules and Actions Scripts to a common set of documentation that was selectively managed via the conditional processing mechanism available to Docs - in much the same way as we did with the B2C and B2B best practice cookbook guides.